src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/LogsTest.java

```diff
@@ -57,59 +61,69 @@ public void checkLogsPage() {
             equalTo("sqlline.log")))
         .body("lastModified", hasItem(containsString(df.format(new Date()))));
   }
+
   @Test
-  public void DrillbitLogContent(){
+  public void DrillbitLogContent() {
     given()
-            .when()
-            .get("/log/drillbit.log/content.json")
-            .then()
-            .statusCode(200)
-            .body("name",equalTo("drillbit.log"))
-            .body(containsString("lines"))
-            .body("maxLines", equalTo(10000));
+        .filter(sessionFilter)
+        .when()
+        .get("/log/drillbit.log/content.json")
+        .then()
+        .statusCode(200)
+        .body("name", equalTo("drillbit.log"))
+        .body(containsString("lines"))
+        .body("maxLines", equalTo(10000));
   }
+
   @Test
-  public void DrillbitOutContent(){
-      given()
-            .when()
-            .get("/log/drillbit.out/content.json")
-            .then()
-            .statusCode(200)
-            .body("name",equalTo("drillbit.out"))
-            .body(containsString("lines"))
-            .body("maxLines", equalTo(10000));
+  public void DrillbitOutContent() {
+    given()
+        .filter(sessionFilter)
+        .when()
+        .get("/log/drillbit.out/content.json")
+        .then()
+        .statusCode(200)
+        .body("name", equalTo("drillbit.out"))
+        .body(containsString("lines"))
+        .body("maxLines", equalTo(10000));
   }
+
   @Test
-  public void DrillbitQueriesContent(){
-      given()
-            .when()
-            .get("/log/drillbit_queries.json/content.json")
-            .then()
-            .statusCode(200)
-            .body("name",equalTo("drillbit_queries.json"))
-            .body(containsString("lines"))
-            .body("maxLines", equalTo(10000));
+  public void DrillbitQueriesContent() {
+    given()
+        .filter(sessionFilter)
+        .when()
+        .get("/log/drillbit_queries.json/content.json")
+        .then()
+        .statusCode(200)
+        .body("name", equalTo("drillbit_queries.json"))
+        .body(containsString("lines"))
+        .body("maxLines", equalTo(10000));
   }
+
   @Test
-  public void SqllineContent(){
-      given()
-            .when()
-            .get("/log/sqlline.log/content.json")
-            .then()
-            .statusCode(200)
-            .body("name",equalTo("sqlline.log"))
-            .body(containsString("lines"))
-            .body("maxLines", equalTo(10000));
+  public void SqllineContent() {
```

Comment:
Start method name from the lower case, please.

Reply:
Done